test/gcr.js

```diff
@@ -84,7 +94,7 @@ test('build and run with clone', (t) => {
   t.ok(build.hasOwnProperty('opts'), 'hasOwnProperty(opts)')
   t.ok(build.hasOwnProperty('output'), 'hasOwnProperty(output)')
   t.ok(build.hasOwnProperty('projectDir'), 'hasOwnProperty(projectDir)')
-  t.equal(build.projectDir, '/tmp/gcr-builds/project-1')
+  t.equal(build.projectDir, path.join('/', 'tmp', 'gcr-builds', 'project-1'))
```

Comment:
This fixes the tests having the path seperator `/` hard coded. Windows can pass this no problem but its returned paths use `\`